lib/anagram_cases.rb

```diff
@@ -0,0 +1,24 @@
+class AnagramCase < OpenStruct
+  def test_name
+    'test_%s' % description.gsub(/[ -]/, '_')
+  end
+
+  def work_load
+    actual = "Anagram.new(\"#{subject}\").match(#{candidates})"
+    "assert_equal #{expected}, #{actual}"
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  def show_comment
+    comment.nil? ? '' : "# #{comment}\n\t\t"
```

Comment:
`\t` is the wrong character to be using.
Ideally there would be no formatting characters in this file.
Perhaps you can find a solution that leaves the formatting in the template file and the logic in this file?